### PR TITLE
chore(dependencies): Update to stryker-api 0.9

### PR DESCRIPTION
**Pull request by mshogren**

### CODE REVIEW

File: test/integration/JestTestRunner.it.ts [hunk 1]
```diff
@@ -138,9 +150,9 @@ describe('JestTestRunner', function () {
         before(() => {
             const testRunnerOptions = {
                 files: [
-                    { path: 'testResources/sampleProject/src/Add.js', mutated: true, included: true },
-                    { path: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false, included: true },
-                    { path: 'testResources/sampleProject/src/Error.js', mutated: false, included: false }],
+                    file({ name: 'testResources/sampleProject/src/Add.js', mutated: true }),
+                    file({ name: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false }),
+                    file({ name: 'testResources/sampleProject/src/Error.js', mutated: false })],
```
Comment (mthmulders) [inline test/integration/JestTestRunner.it.ts]: I think this one should have `included: false`, if I'm not mistaken?

**Comment by mthmulders** [comment]
Thanks for your contribution!

File: test/integration/JestTestRunner.it.ts (hunk 1 above)
Comment (mshogren) [reply test/integration/JestTestRunner.it.ts]: You are definitely correct

**Comment by mthmulders** [comment]
I suppose `bb16acc` slipped in because #18 was already merged?

**Comment by mshogren** [comment]
Sorry about that extra commit.  I am still getting the hang of working on multiple pull requests at once.  At work we are still stuck with svn.

**Pull request merged**